test/test_utils.py

```diff
@@ -54,6 +55,9 @@
 load_tests = load_tests  # noqa: PLW0127
 
 HAS_CUDA = torch.cuda.is_available()
+HAS_XPU = torch.xpu.is_available()
+HAS_GPU = HAS_CUDA or HAS_XPU
+device_type = get_gpu_type()
```

Comment:
```suggestion
device_type = acc.type if (acc := torch.accelerator.current_accelerator(True)) else "cpu"
```
It is not reasonable to make this UT depend on Inductor utils.